src/04-modality-specific-files/01-magnetic-resonance-imaging-data.md

```diff
@@ -207,6 +207,26 @@ entity corresponds to modality suffix,
 such as `T1w` or `inplaneT1`, referenced by the defacemask image.
 For example, `sub-01_mod-T1w_defacemask.nii.gz`.
 
+The OPTIONAL [`task-<label>`](../99-appendices/entities.md#task) entity can be used
```

Comment:
```suggestion
The OPTIONAL [`task-<label>`](../appendices/entities.md#task) entity can be used
```

Sorry. I was too quick. We also removed the prefix from the folder name, hence this doc building is still failing... 🙈

Reply:
Ok, maybe it's easier if I re-update my fork as well?

Reply:
yup this may help too. 😄